src/utcoffset.jl

```diff
@@ -57,8 +57,17 @@ function offset_string(offset::UTCOffset, iso8601::Bool=false)
 end
 
 Base.print(io::IO, o::UTCOffset) = print(io, offset_string(o, true))
+
 function Base.show(io::IO, o::UTCOffset)
-    # Show DST as a separate offset since we want to distinguish between normal hourly
-    # daylight saving time offsets and exotic DST offsets (e.g. midsummer time).
-    print(io, "UTC", offset_string(o.std), "/", offset_string(o.dst))
+    if get(io, :compact, false)
+        # Show DST as a separate offset since we want to distinguish between normal hourly
+        # daylight saving time offsets and exotic DST offsets (e.g. midsummer time).
+        print(io, "UTC", offset_string(o.std), "/", offset_string(o.dst))
+    else
+        invoke(show, Tuple{IO, Any}, io, o)
```

Comment:
Seems a little weird that your invoking the default show, might be good to include a comment.

Reply:
Same goes for elsewhere.